instancetypes/cx/1/cx1.yaml

```
@@ -15,6 +15,16 @@ metadata:
       the IO threading from cores dedicated to the workload.
       In addition, in this series, the NUMA topology of the used
       cores is provided to the VM.
+
+      Number of spec.cpu.maxSockets is set to the same number as of
+      guest cpus. This is a preventative measure. It avoids
```

```suggestion
      guest cpus. This avoids
```

I'd drop that sentence in the middle.